xml/gl.xml

```diff
@@ -79,7 +79,18 @@ typedef unsigned int GLhandleARB;
         <type>typedef void (<apientry/> *<name>GLVULKANPROCNV</name>)(void);</type>
     </types>
 
-    <!-- SECTION: GL parameter class type definitions. -->
+    <!-- SECTION: GL parameter class type definitions.
+         The groups are intended to contain all the possible legal values
+         for corresponding function parameters, but it is likely that many
+         of the groups are out of date relative to current OpenGL and OpenGL
+         ES specifications, and the many extensions to those specifications.
+         As such, they may not be a reliable source for enumeration info.
+
+         We welcome assistance from the community in making the enum groups
+         more complete. Khronos does not use the enum group information
+         ourselves, and the OpenGL Working Group does not have internal
```

Comment:
Grammatical nitpick :|
```suggestion
         themselves, and the OpenGL Working Group does not have internal
```

Reply:
I just dropped the word, rather than open up to more potential grammar issues.